augur/application/db/util.py

```diff
@@ -43,48 +53,47 @@ def execute_session_query(query, query_type="all"):
     return catch_operational_error(func)
 
 
-
-def convert_orm_list_to_dict_list(result):
-    new_list = []
+def convert_orm_list_to_dict_list(result: List[Any]) -> List[Dict[str, Any]]:
+    """
+    Converts a list of ORM model instances to a list of dictionaries.
+    """
+    new_list: List[Dict[str, Any]] = []
 
     for row in result:
-        row_dict = row.__dict__
-        try:
-            del row_dict['_sa_instance_state']
-        except:
-            pass
-        
+        row_dict = dict(row.__dict__)  # Copy to avoid mutating the ORM instance
+        row_dict.pop("_sa_instance_state", None)  # Remove SQLAlchemy internal state
         new_list.append(row_dict)
-    
-    return new_list
 
+    return new_list
 
 
-def convert_type_of_value(config_dict, logger=None):
-        
-    data_type = config_dict["type"]
+def convert_type_of_value(
+    config_dict: Dict[str, Any], logger: Optional[Any] = None
```

Comment:
does config_dict come from somewhere? could its type be specified more precisely than `Any`? same with  the logger